frontend/src/components/Header.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div>
+    <v-app-bar class="navbar-custom" dark clipped-right color="#2C3E50">
+      <v-toolbar-title>
+        <a href="https://codeforpoznan.pl/">
+          <v-img
+            :src="image"
+            contain
+            max-height="20"
+            aspect-ratio="1.7"
+            max-width="500"
+            min-width="350"
+            position="left"
+          ></v-img>
+        </a>
+      </v-toolbar-title>
+      <v-spacer></v-spacer>
+      <v-tabs
+        background-color="transparent"
+        right
+        class="hidden-sm-and-down"
+        mobile-break-point="768px"
+      >
+        <template v-for="item in items">
+          <v-tab align-right @click="$vuetify.goTo(item.id)" :key="item.id">{{
```

Comment:
The content within this should be centered so that in mobile view it is not aligned to the left. Unless you think it's ok and it shouldn't be centered. I'm not that sure about either but how it is now seems to me a lil bit iffy.